velox/core/PlanNode.h

```diff
@@ -2721,6 +2723,10 @@ class PartitionedOutputNode : public PlanNode {
     return outputType_;
   }
 
+  const bool isRootFragment() const {
```

Comment:
no need to use const since the return is a scalar value here.
`bool isRootFragment() const {`
Similar to below
`bool isPartitioned() const {`